rest-service/manager_rest/rest/resources_v3_1/searches.py

```diff
@@ -0,0 +1,84 @@
+from flask import request
+
+from manager_rest.security import SecuredResource
+from manager_rest.security.authorization import authorize
+from manager_rest.storage import models, get_storage_manager
+
+from ..responses_v2 import ListResponse
+from .. import rest_decorators, rest_utils
+from ..filters_utils import (create_filter_rules_list,
+                             get_filter_rules_from_filter_id)
+
+
+class ResourceSearches(SecuredResource):
+    def post(self, resource_model, _include, filters, pagination, sort,
```

Comment:
Why is this a post when it is not modifying state on the manager? It should be a get, surely?